lib/tests/test_wallet_vertical.py

```diff
@@ -280,3 +281,26 @@ def test_bip39_multisig_seed_p2sh_segwit(self, mock_write):
 
         self.assertEqual(w.get_receiving_addresses()[0], '35LeC45QgCVeRor1tJD6LiDgPbybBXisns')
         self.assertEqual(w.get_change_addresses()[0], '39RhtDchc6igmx5tyoimhojFL1ZbQBrXa6')
+
+    @mock.patch.object(storage.WalletStorage, '_write')
+    def test_bip39_multisig_seed_p2sh_segwit_testnet(self, mock_write):
+        constants.set_testnet()
```

Comment:
This is not really OK in general because tests might run in parallel.
If you try to do this in e.g. test_bitcoin.py, there will be dozens of failures.
I'll do a bit of refactoring in a few hours and fix this though, so just leave it.

Reply:
Ah yes I had a feeling I was being a bit hacky with that, cheers :)